pkg/epp/scheduling/config.go

```diff
@@ -49,7 +49,7 @@ type SchedulerConfig struct {
 // For build time plugins changes, it's recommended to change the defaultConfig variable in this file.
 var defaultConfig = &SchedulerConfig{
 	preSchedulePlugins:  []plugins.PreSchedule{},
-	filters:             []plugins.Filter{&filter.SheddableRequestFilter{}, filter.LowLatencyFilter},
```

Comment:
@liu-cong You made things very easy for me with the filter cleanup you did. Thanks!

Reply:
Also had to rebase onto Nir's change. Same comment applies though. Thanks all!